charts/openstack-cloud-controller-manager/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Openstack Cloud Controller Manager Helm Chart
 icon: https://object-storage-ca-ymq-1.vexxhost.net/swift/v1/6e4619c416ff4bd19e1c087f27a43eea/www-images-prod/openstack-logo/OpenStack-Logo-Vertical.png
 home: https://github.com/kubernetes/cloud-provider-openstack
 name: openstack-cloud-controller-manager
-version: 1.1.1
+version: 1.2.0
```

Comment:
why bump to 1.2? maybe need 1.1.2?

Reply:
Hi, thanks for getting back to me so quickly.

I was following [Semantic versioning](https://semver.org/), which [should be used for Helm charts](https://helm.sh/docs/chart_best_practices/conventions/#version-numbers):

> Given a version number MAJOR.MINOR.PATCH, increment the:
>   MAJOR version when you make incompatible API changes,
>   MINOR version when you add functionality in a backwards compatible manner, and
>   PATCH version when you make backwards compatible bug fixes.

Reply:
Hi both, appreciate the discussion here. However, it has been decided (within the team) to only bump the chart version when we do release, so please still keep `1.1.1`.

@jichenjc remember? Please see the discussion in Slack. I just need to add some doc for contributors.

Otherwise, this PR looks good to me.